src/sass/layouts/_header.scss

```diff
@@ -0,0 +1,212 @@
+.header {
+  backdrop-filter: blur(1rem);
+  display: flex;
+  position: fixed;
+  z-index: 100;
+  max-height: 8rem;
+  width: 100%;
+  padding: 2rem 4rem;
+  justify-content: space-between;
+  align-items: center;
+  background-image: linear-gradient(
+    to right,
+    rgb(55 65 81 / 30%),
+    rgb(55 65 81 / 60%)
+  );
+
+  @include respond(tab-port) {
+    padding: 2rem 3rem;
+  }
+
+  @include respond(phone) {
+    padding: 2rem;
+  }
+
+  &__logo-container {
+    flex: 0 0 5%;
+    min-width: 5.5rem;
+
+    &__img {
+      height: 100%;
+      max-width: 100%;
+    }
+  }
+
+  &__btn-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    list-style: none;
+
+    @include respond(phone) {
+      flex-direction: column;
+      justify-content: center;
+      height: 100vh;
+      width: 60%;
+      position: absolute;
+      left: 100%;
+      top: 0;
+      z-index: 101;
+      background-image: linear-gradient(to right, rgb(55 65 81), rgb(55 65 81));
+      transition: all 0.5s;
+      visibility: hidden;
+      opacity: 0;
```

Comment:
what is the diff b/w these two properties?

Reply:
Visibility simply toggles the element between visible, hidden or collapse (if table) while opacity let us set a range from 0 to 1 allowing a transparent effect.

Also, elements with visibility set to hidden will not be interactive while the elements with opacity set to 0 will still be interactive but not visible.

Reply:
but why you used both of them here?

Reply:
I used visibility to make sure the hamburger menu is not interactive when hidden and opacity to give it a fade-in effect.
But in the recent changes that I made, I removed the opacity as the fade in effect was not looking good with the left slide effect.